pylcm/__init__.py

```diff
@@ -139,7 +141,7 @@ def msg_to_dict(  # noqa: C901
                 rgb_image = my_value[0].data
             depth_data = zlib.decompress(my_value[1].data)
             depth_image = np.frombuffer(depth_data, dtype="uint16").reshape(
-                480, 640
+                image_dimensions[1], image_dimensions[0]
```

Comment:
```
depth_image = np.frombuffer(depth_data, dtype="uint16").reshape(image_dimensions)
```
no need to unpack. even if unpacking is needed, you can use asterisks

Reply:
I wanted to flip the order though, since I think it makes sense to pass param as (640, 480) but we need to reshape to (480, 640)

Reply:
well don't flip and just ask the user to pass (480, 640) as that is the convention for matrix dimensions and 2D images in computing (HxW, contrary to the usual WxH in the media, I know).

In the future if you need to flip an array, simply do `array[::-1]`